ts/parts/DataLabels.ts

```diff
@@ -2180,7 +2181,20 @@ if (seriesTypes.column) {
         );
 
         // If label was justified and we have contrast, set it:
-        if (point.isLabelJustified && point.contrastColor) {
+        if (
+            point.contrastColor &&
+            point.shapeArgs &&
+            intersectRect(
```

Comment:
It's probably better to refactor the function to take two objects, each having `{ x, y, width, height }`, so we don't have to pass over eight nested parameters.

Reply:
Then it is actually better to have two parameter, because we have interfaces like `PositionObject` and `DimensionObject`.

Reply:
Even better, we have the `BBoxObject` interface, that takes all four properties.